dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/resource/FileResourcesMetadataController.java

```diff
@@ -118,6 +128,24 @@ public ResponseEntity save(@RequestBody FileResource fileResource)
     }
   }
 
+  @GetMapping("/api/fileresources/images")
+  @ResponseBody
+  public SearchPageResponse<FileResource> search(
+      @RequestParam(name = "nullHandling", required = false, defaultValue = "NATIVE")
+          NullHandling nullHandling,
+      @RequestParam(name = "pageNumber", required = false, defaultValue = "0") int pageNumber,
+      @RequestParam(name = "pageSize", required = false, defaultValue = "5") int pageSize,
+      @RequestParam(name = "sortField", required = false, defaultValue = "uuid") String sortField,
```

Comment:
Warum wird default-mäßig nach der UUID sortiert? Die ist doch random? Wäre es nicht sinniger als Sortier-Kriterium etwas zu nehmen wie "zuletzt erstellt" oder "wird in so vielen dokumenten verwendet"?

Reply:
Da hast du recht, das macht recht wenig Sinn so. Ich hätte z.B. `created` oder `lastModified`, nach denen ich sortieren könnte, welches wäre für dich da sinnvoller? Ich tendiere ja zu `lastModified`.

Reply:
ja, ich denke auch dass `lastModified` wahrscheinlich sinnvoller ist :-) Das dürfte ja auch weitestgehend identisch sein zu `created`, oder?

Reply:
In der Regel schon, ja - sobald jemand die `FileResource` ändert, wird es eben aktualisiert.